### PR TITLE
Fix handling of minimum operations, and forward rotate/config requests to Primary

### PR DESCRIPTION
**Pull request by sgmiller**

### CODE REVIEW

**Review by briankassouf — APPROVED** [review]
👍

File: vault/keyring.go [hunk 1]
```diff
@@ -253,9 +253,12 @@ func (c KeyRotationConfig) Clone() KeyRotationConfig {
 }
 
 func (c *KeyRotationConfig) Sanitize() {
-	if c.MaxOperations == 0 || c.MaxOperations > absoluteOperationMaximum || c.MaxOperations < absoluteOperationMinimum {
+	if c.MaxOperations == 0 || c.MaxOperations > absoluteOperationMaximum {
 		c.MaxOperations = absoluteOperationMaximum
 	}
+	if c.MaxOperations < absoluteOperationMinimum {
+		c.MaxOperations = absoluteOperationMinimum
```
Comment (HridoyRoy) [inline vault/keyring.go:260]: Could we just skip this statement, so c.MaxOperations remains the same as it was previously?

**Review by HridoyRoy — CHANGES_REQUESTED** [review]

File: vault/keyring.go (hunk 1 above)
Comment (sgmiller) [reply vault/keyring.go:260]: no, this is making sure it's never below the min.

Comment (sgmiller) [reply vault/keyring.go:260]: oh, in the API call we could, yes.

Comment (sgmiller) [reply vault/keyring.go:260]: Updated

**Review by HridoyRoy — APPROVED** [review]

**Pull request merged**